README.md

```diff
@@ -28,4 +28,11 @@ render the docs. Please give the docs time to render, they require external asse
 ```
 make
 apiary preview --path="apiary.apib"
-````
+```
+
+or
+```
+make
```

Comment:
When would this be used as opposed to the commands above? Please add a bit of explanation to give it context.